testgrid/updater/main.go

```diff
@@ -17,12 +17,27 @@ limitations under the License.
 package main
 
 import (
+	"context"
 	"fmt"
+	"log"
 
 	"k8s.io/test-infra/testgrid/updater/state"
+
+	"cloud.google.com/go/storage"
 )
 
 func main() {
+	ctx := context.Background()
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		log.Fatalf("Failed to create storage client: %v", err)
+	}
+	bkt := client.Bucket("kubernetes-jenkins")
```

Comment:
should the bucket be configurable, rather than hard-coded like this?

Reply:
Uh, no.

Reply:
ok, I clearly don't understand what this is used for. :)

Reply:
This code that is being expanded in small commits. The fact that it doesn't *do* anything just yet means it's all implicitly `//TODO`. :-)

Reply:
I have more commits to add, but I'd rather get in the vendored code first so the rest are vastly easier to review. And this at least uses it so dep won't remove it